### ISSUE TITLE
Convert standard date format to Julian date and vice-versa

### ISSUE THREAD

**Issue by magar51** [codinasion-archive/codinasion-monorepo#5402]
### Description

A Julian date is a way of displaying a date that combines the current year with the number of days since the beginning of the year. For example, January 1, 2007 is 2007001, and December 31, 2007 is 2007365.

The first two digits of a Julian date represent the year, and the last three digits represent the number of days since the beginning of the year. For example, 21-001 represents January 1, 2021, and 22-165 represents June 14, 2022.

### Example

```
Input  : 2023-06-10
Output : 2023160
```

**Comment by isyuricunha** [comment]
Hi,

You need to include the ID of the solution you started working on. In this case, you created #5402, right? So you need to put its ID in the body of the PR. For example:

```
 Description
 Related Issue
Closes #5402
```

Do you understand?

I will add the ID this time, but please remember to do it next time. With that said, let's wait for @harshraj8843 and @anandfresh to take another look at this PR and the issue.

**Comment by harshraj8843** [comment]
Program added successfully :tada:

Thanks for your contribution :hugs:
